tests/Unit/Core/ViewHelper/AbstractTagBasedViewHelperTest.php

```diff
@@ -153,6 +153,26 @@ public function testValidateAdditionalArgumentsThrowsExceptionIfContainingNonDat
         $viewHelper->validateAdditionalArguments(['foo' => 'bar']);
     }
 
+    /**
+     * @test
+     */
+    public function handleAdditionalArgumentsSetsTagAttributesForAriaArguments()
+    {
+        $viewHelper = $this->getAccessibleMock(
+            AbstractTagBasedViewHelper::class,
+            ['dummy'],
+            [],
+            '',
+            false
+        );
+        $viewHelper->setRenderingContext(new RenderingContextFixture());
+        $tagBuilder = $this->getMock(TagBuilder::class, ['addAttribute']);
+        $tagBuilder->expects($this->at(0))->method('addAttribute')->with('aria-labelledby', 'foo');
+        $tagBuilder->expects($this->at(1))->method('addAttribute')->with('aria-live', 'polite');
+        $viewHelper->setTagBuilder($tagBuilder);
+        $viewHelper->handleAdditionalArguments(['aria-labelledby' => 'foo', 'aria-live' => 'polite']);
```

Comment:
`$viewHelper->initializeArgumentsAndRender();` must be added here to fix the test.